Purchases/Caching/RCDeviceCache.m

```diff
@@ -163,8 +181,9 @@ - (void)cacheOfferings:(RCOfferings *)offerings {
     [self.offeringsCachedObject cacheInstance:offerings];
 }
 
-- (BOOL)isOfferingsCacheStale {
-    return self.offeringsCachedObject.isCacheStale;
+- (BOOL)isOfferingsCacheStaleWithIsAppBackgrounded:(BOOL)isAppBackgrounded {
```

Comment:
having a different cache might not be strictly necessary for offerings, really - it's unlikely that you'd need offerings while the app is running in the background. can't hurt, though.